zone/mob.h

```diff
@@ -1253,18 +1338,35 @@ class Mob : public Entity {
 	virtual int			GetStuckBehavior() const { return 0; }
 
 	void				NPCSpecialAttacks(const char* parse, int permtag, bool reset = true, bool remove = false);
-	inline uint32		DontHealMeBefore() const { return pDontHealMeBefore; }
-	inline uint32		DontBuffMeBefore() const { return pDontBuffMeBefore; }
-	inline uint32		DontDotMeBefore() const { return pDontDotMeBefore; }
-	inline uint32		DontRootMeBefore() const { return pDontRootMeBefore; }
-	inline uint32		DontSnareMeBefore() const { return pDontSnareMeBefore; }
-	inline uint32		DontCureMeBefore() const { return pDontCureMeBefore; }
-	void				SetDontRootMeBefore(uint32 time) { pDontRootMeBefore = time; }
-	void				SetDontHealMeBefore(uint32 time) { pDontHealMeBefore = time; }
-	void				SetDontBuffMeBefore(uint32 time) { pDontBuffMeBefore = time; }
-	void				SetDontDotMeBefore(uint32 time) { pDontDotMeBefore = time; }
-	void				SetDontSnareMeBefore(uint32 time) { pDontSnareMeBefore = time; }
-	void				SetDontCureMeBefore(uint32 time) { pDontCureMeBefore = time; }
+	inline uint32		DontHealMeBefore() const { return m_dont_heal_me_before; }
+	inline uint32		DontGroupHealMeBefore() const { return m_dont_group_heal_me_before; }
+	inline uint32		DontGroupHoTHealMeBefore() const { return m_dont_group_hot_heal_me_before; }
+	inline uint32		DontRegularHealMeBefore() const { return m_dont_regular_heal_me_before; }
+	inline uint32		DontVeryFastHealMeBefore() const { return m_dont_very_fast_heal_me_before; }
+	inline uint32		DontFastHealMeBefore() const { return m_dont_fast_heal_me_before; }
+	inline uint32		DontCompleteHealMeBefore() const { return m_dont_complete_heal_me_before; }
+	inline uint32		DontGroupCompleteHealMeBefore() const { return m_dont_group_complete_heal_me_before; }
+	inline uint32		DontHotHealMeBefore() const { return m_dont_hot_heal_me_before; }
+	inline uint32		DontBuffMeBefore() const { return m_dont_buff_me_before; }
+	inline uint32		DontDotMeBefore() const { return m_dont_dot_me_before; }
+	inline uint32		DontRootMeBefore() const { return m_dont_root_me_before; }
+	inline uint32		DontSnareMeBefore() const { return m_dont_snare_me_before; }
+	inline uint32		DontCureMeBefore() const { return m_dont_cure_me_before; }
```

Comment:
A lot of this looks very bot AI specific, which should be in bots if that's the case. Same with the member variables.

I also don't see a lot of this being used

![image](https://github.com/user-attachments/assets/2f951e3f-0182-4916-a8e7-0d99db74c6f0)

Reply:
Removed the unused, remaining are mainly mob and merc